orangecontrib/text/widgets/owsentimentanalysis.py

```diff
@@ -0,0 +1,75 @@
+from AnyQt.QtWidgets import QApplication
+
+from Orange.widgets import gui, settings
+from Orange.widgets.utils.signals import Input, Output
+from Orange.widgets.widget import OWWidget
+from orangecontrib.text import Corpus
+from orangecontrib.text.sentiment import Vader_Sentiment, Liu_Hu_Sentiment
+
+METHODS = [Liu_Hu_Sentiment(),
+           Vader_Sentiment()]
+
+class OWSentimentAnalysis(OWWidget):
+    name = "Sentiment Analysis"
+    description = "Predict sentiment from text."
+    icon = "icons/SentimentAnalysis.svg"
+    priority = 30001
```

Comment:
This has already been discussed - priority numbering will be addressed in another issue, where the range will be updated to hundreds. It just makes a lot more sense than having randomly high priority numbers. :)